index.html

```diff
@@ -33,6 +33,10 @@
         <img class="navimg" src="img/icons/file-earmark-arrow-down.svg"/>
         Export
       </button>
+      <label class="btn btn-outline-primary">
+        <input type="checkbox" id='checkbox_min_max' checked=true v-on:click="toggleMinMaxButtonClicked"> min/avg/max
```

Comment:
Since, avg is always shown with this button, we can remove it from the description text.

Reply:
The glue between the checkbox element and the JS code should probably be `v-model` or `v-bind`. I'm just not sure how this can deal gracefully with the intermediate state (see https://github.com/vuejs/vue/issues/4094).